federation/pkg/federation-controller/replicaset/planner/planner.go

```diff
@@ -152,9 +151,7 @@ func (p *Planner) Plan(replicasToDistribute int64, availableClusters []string, c
 	// TODO: Prove that clusterCount * log(replicas) iterations solves the problem or adjust the number.
 	// TODO: This algorithm is O(clusterCount^2 * log(replicas)) which is good for up to 100 clusters.
```

Comment:
This comment is no longer valid.
We will keep on iterating as long as we did some modification (i.e we set modified = true).